### ISSUE TITLE
\hline is not showing with vertical addition(IOS)

### ISSUE THREAD

**Issue by NounViraneath** [ShaMan123/react-native-math-view#45]
I am trying to do something like this with react-native-math-view.
`$$\begin{array}{r} 735 \\\ +196 \\\ \hline 931 \end{array}$$`

But \hline is not showing on my IOS emulator.
![image](https://user-images.githubusercontent.com/6902893/118362635-749e2d80-b5ba-11eb-8ed3-adea2b99fe5b.png)

And, It's fine with Android device...

**Comment by ShaMan123** [comment]
Please file a [bug report](https://github.com/ShaMan123/react-native-math-view/issues/new?assignees=&labels=&template=bug_report.md&title=) with all information.

**Comment by ShaMan123** [comment]
There's a reason why there's the template.
Following solves most of the problems.
Your latex has an error which is easily debugged by going through the template.
This is correct: `$$\begin{array}{r} 735 \\ +196 \\ \hline 931 \end{array}$$`

**Comment by NounViraneath** [comment]
thanks for your comments.

`$$\begin{array}{r} 735 \\ +196 \\ \hline 931 \end{array}$$`
with the above latex, `\hline` is still not showing in my ios emulator.

I am using
react-native-math-view@^3.8.2
mathjax-full@^3.1.2

**Comment by ShaMan123** [comment]
Try using the fallback component on android:
 `import MathViewFallback, { MathText } from 'react-native-math-view/src/fallback';` 
If the bug persists it means the bug originates from `mathjax` or `react-native-svg`.

It seems that the line has a css property attached to it by mathjax which can explain why it doesn't show.
Take a look https://codesandbox.io/s/zealous-kilby-snd7u?file=/index.html
and inspect the line.
If you add the missing css to the defined class it shows.
So I'm guessing that the issue - css isn't inlined in the svg output so it doesn't exist when parsing.
```css
line.mjx-solid {
    /* color: black; */
    /* stroke-width: 3; */
    stroke-width: 70px;
}
```

**Comment by ShaMan123** [comment]
https://github.com/mathjax/MathJax-src/blob/master/ts/util/StyleList.ts
https://github.com/mathjax/MathJax-src/blob/master/ts/util/Styles.ts

**Comment by ShaMan123** [comment]
This is the reason: https://github.com/react-native-svg/react-native-svg/issues/260

**Comment by ShaMan123** [comment]
fixed 3.9.1

**Comment by NounViraneath** [comment]
Thanks for the solutions!
I have tried to reinstall with the latest version react-native-math-view@3.9.1
but I got mathjax-full@3.1.4 compile error
```
....
> mathjax-full@3.1.4 postcompile
> npm run --silent copy:mj2

node:internal/fs/utils:320
    throw err;

Error: ENOENT: no such file or directory....
path: 'node_modules/mathjax-full/ts/input/tex/mhchem/mhchem_parser.js',
dest: '/node_modules/mathjax-full/js/input/tex/mhchem/mhchem_parser.js' 
```

**Comment by NounViraneath** [comment]
I have tried to change  `innerPathToFile` in scripts/compileMathjax.js.
`const innerPathToFile = 'input/tex/mhchem/mhchemparser/mhchemParser.js';`
And, it seems to work well with the latest version.
\hline is also showing on IOS.

**Comment by ShaMan123** [comment]
I have disabled the script `scripts/compileMathjax.js` in v3.9.2
The error that surfaces is a mathjax postinstall error.
Could you remove `mathjax-full` and re-install it after upgrade `react-native-math-view` and see if the error persists?
